_docs/quickstart.md

```diff
@@ -41,19 +41,21 @@ $ cd flux-core
 ```
 
 Build flux-core. In order to build python bindings ensure you have
-python-2.7 and python-cffi available in your current environment:
+python-3.6 and python-cffi available in your current environment:
 
 ```console
-$ module load python/2.7 python-cffi python-pycparser 
-$ ./autogen.sh && ./configure 
-Running aclocal ... 
-Running libtoolize ... 
-Running autoheader ... 
+$ module load python/3.6 python-cffi python-pycparser
+$ ./autogen.sh && ./configure
+Running aclocal ...
+Running libtoolize ...
+Running autoheader ...
 ...
 $ make -j 8
 ...
 ```
 
+_Note: Flux still supports python-2.7, but we recommend users to use python-3.6 or higher as the Python community will stop to maintain this version in 2020._
+
 Ensure all is right with the world by running the built-in `make check`
```

Comment:
Consider updating the `make check` output below (since it references a very old version number), or just remove it all together so we don't have to keep updating it.